install/kubernetes/helm/istio-remote/charts/security/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ServiceAccount
+{{- if .Values.global.imagePullSecrets }}
+imagePullSecrets:
+{{ toYaml .Values.global.imagePullSecrets | indent 2 }}
+{{- end }}
+metadata:
+  name: istio-citadel-service-account
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "security.name" . }}
```

Comment:
Please use the actual names. 

Also make sure it matches the CA in the main templates in terms of labels - it is hard to upgrade or move back and forth.

Reply:
this is a complete copy and paste form main template.  For the time being, plan to c&p as the master istio security template changes.